src/corefx/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+
+project(System.Security.Cryptography.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+add_definitions(-DBIT64=1)
+
+find_library(CRYPTO NAMES crypto)
+if(CRYPTO STREQUAL CRYPTO-NOTFOUND)
+    message(WARNING "Cannot find libcrypto, skipping build for System.Security.Cryptography.Native. .NET cryptography is not expected to function. Try installing openssl-dev (or the appropriate package for your platform)")
```

Comment:
Is this the right name of the package? I am not able to get it on my Ubuntu. Maybe it should be libssl-dev?